src/ElmFormat/Execute.hs

```diff
@@ -22,9 +23,19 @@ forHuman operation =
 
 
 {-| Execute Operations in a fashion appropriate for use by automated scripts. -}
-forMachine :: ElmVersion -> OperationF a -> IO a
+forMachine :: ElmVersion -> OperationF a -> StateT Bool IO a
 forMachine elmVersion operation =
     case operation of
-        InFileStore op -> FileStore.execute op
+        InFileStore op -> lift $ FileStore.execute op
         InInfoFormatter op -> Json.format elmVersion op
-        DeprecatedIO io -> io
+        DeprecatedIO io -> lift io
+
+
+forMachineInit :: (IO (), Bool)
```

Comment:
:art: It wasn't apparent to me that the state is for `printComma` (right?) -> A comment? I'm not sure of the volatility of the code around here so leaving it up to youl